src/legacy/ui/public/state_management/state.js

```diff
@@ -316,6 +316,27 @@ export function StateProvider(Private, $rootScope, $location, stateManagementCon
     return this._urlParam;
   };
 
+  /**
+   * Returns an object with each property name and value corresponding to the entries in this collection
+   * excluding fields started from '$', '_' and all methods
+   *
+   * @return {object}
+   */
+  State.prototype.toObject = function () {
```

Comment:
Is this related to the deletions?
If not, what is this related to?

Reply:
Please see remove src/legacy/ui/public/utils/base_object.ts file

Reply:
@lizozom sorry for not detailed answer. Please see next diagram 

![Untitled Diagram (1)](https://user-images.githubusercontent.com/20072247/70507415-5f0f1380-1b3d-11ea-80f0-81180fee9404.jpg)

Reply:
That was a great explanation